common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -2922,6 +2920,10 @@ def _copy_subdag(self, user_id, destination_version, block_key, source_blocks, d
                 raw=True,
                 block_defaults=new_block.defaults
             )
+            # _new_block doesn't copy the original_usage data needed for library blocks:
+            if new_block.edit_info.original_usage:
+                destination_block.edit_info.original_usage = new_block.edit_info.original_usage
+                destination_block.edit_info.original_usage_version = new_block.edit_info.original_usage_version
```

Comment:
Yeah, rather than having to manually copy `original_usage` after every call to `_new_block`, it would be better to make '_new_block` preserve it, if it exists.